.gitignore

```diff
@@ -0,0 +1,89 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Please remove this file. These are not specific to this project, they belong in a global `.gitignore`. Also this isn't related to this change.